site/content/news/hacktoberfest.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Join us for Hacktoberfest 2020"
+date: 2020-09-28T09:00:00+10:00
+layout: "single"
+categories: ["news"]
+---
+
+<a href="https://hacktoberfest.digitalocean.com/">
+<img align="right" alt="Hactoberfest 2020" src="https://hacktoberfest.digitalocean.com/assets/HF-full-logo-b05d5eb32b3f3ecc9b2240526104cf4da3187b8b61963dd9042fdc2536e4a76c.svg"
+width=300></a>
+
+Hacktoberfest is a yearly event, a celebration of open source software, put together by DigitalOcean, to encourage people to start contributing to open source software. We would like to encourage you to [sign up and take part](https://hacktoberfest.digitalocean.com/)!
+
+<!--more-->
+
+The idea is that you contribute to an open source project, any open source project, by submitting a Pull Request (PR) to that project. If your PRs are accepted, you can win a very snazzy T-shirt, or if you prefer you can opt to plant a tree instead. See the [Hactoberfest sign up page](https://hacktoberfest.digitalocean.com/) for more details.
```

Comment:
nit-picking here: perhaps use "earn a very snazzy T-shirt"  instead of "win" ? Feels a bit like having a low chance to actually end up with a price.

Reply:
Yeah, good point actually, thanks, will fix!